process_report/process_report.py

```diff
@@ -62,15 +65,42 @@ def is_old_pi(old_pi_dict, pi, invoice_month):
     return False
 
 
+def get_invoice_bucket():
+    s3_vars = load_S3_env_vars()
+    b2_resource = boto3.resource(
+        service_name="s3",
+        endpoint_url=s3_vars["B2_ENDPOINT"],  # Backblaze endpoint
+        aws_access_key_id=s3_vars["B2_KEY_ID"],  # Backblaze keyID
+        aws_secret_access_key=s3_vars["B2_APP_KEY"],  # Backblaze applicationKey
+        config=Config(
+            signature_version="s3v4",
+        ),
+    )
+    return b2_resource.Bucket(s3_vars["B2_BUCKET_NAME"])
+
+
+def load_S3_env_vars() -> dict:
```

Comment:
Read the environment variables directly in the code block above, this is unnecessary.